docs/concepts/workloads/controllers/daemonset.md

```diff
@@ -98,15 +98,43 @@ If you do not specify either, then the DaemonSet controller will create Pods on
 
 ## How Daemon Pods are Scheduled
 
-Normally, the machine that a Pod runs on is selected by the Kubernetes scheduler.  However, Pods
+### Scheduled by DaemonSet controller (default)
+
+Normally, the machine that a Pod runs on is selected by the Kubernetes scheduler. However, Pods
 created by the DaemonSet controller have the machine already selected (`.spec.nodeName` is specified
-when the Pod is created, so it is ignored by the scheduler).  Therefore:
+when the Pod is created, so it is ignored by the scheduler). Therefore:
 
  - The [`unschedulable`](/docs/admin/node/#manual-node-administration) field of a node is not respected
    by the DaemonSet controller.
  - The DaemonSet controller can make Pods even when the scheduler has not been started, which can help cluster
    bootstrap.
 
+### Scheduled by default scheduler (with ScheduleDaemonSetPods alpha feature)
+
+In version 1.10, `ScheduleDaemonSetPods` was introduced as an alpha feature. This features schedules
+ DaemonSet Pods with the default scheduler instead of with the Daemonset controller.
+
+When this feature is enabled, the `NodeAffinity` term (instead of `.spec.nodeName`) is added to the DaemonSet Pods,
+as in the following example. This enables the default scheduler to bind the Pod to the target host. If the `kubernetes.io/hostname`
+ node affinity of DaemonSet Pod already exists, it will be replaced. Those operations will only modify the Pods of DaemonSet,
+ not chagnes are made to the `spec.template` of DaemonSet.
+
+```
+nodeAffinity:
+  requiredDuringSchedulingIgnoredDuringExecution:
+    nodeSelectorTerms:
+    - matchExpressions:
+      - key: kubernetes.io/hostname
+        operator: In
+        values:
+        - target-host-name
+```
+
+**NOTES**: As the DaemonSet pods are scheduled by default scheduler, it'll **NOT** tolerate
+ [`unschedulable`](/docs/admin/node/#manual-node-administration) field of a node when this feature enabled.
```

Comment:
I thought DS will still ignore the `unschedulable` field on nodes?
https://github.com/kubernetes/kubernetes/issues/60163#issuecomment-369445696

@bsalamat

Reply:
We cannot change the behavior of DS controller. DS controller should add tolerations for "NoSchedule" taint and default scheduler should treat that toleration specially for now and ensure that such pods tolerate `unschedule` condition as well. Once we graduate "tainting nodes by condition" to beta, we can remove this special treatment of the toleration in the scheduler.

Reply:
> DS controller should add tolerations for "NoSchedule" taint and default scheduler should treat that toleration specially for now and ensure that such pods tolerate unschedule condition as well.

@bsalamat this means that any pod with "NoSchedule" toleration will ignore node's `unschedulable` field (not taint). Correct?

Reply:
> this means that any pod (including non-Daemon ones) with "NoSchedule" toleration will ignore node's unschedulable field (not taint). Correct?

That's right. However, given #61161, we no longer need to have the scheduler hack that I mentioned above.

Reply:
Yes. I'd like to get #61161 merged and remove this limitation.